com.unity.render-pipelines.core/Runtime/RenderPipeline/IShaderVariantSettings.cs

```diff
@@ -5,11 +5,14 @@ namespace UnityEngine.Rendering
     /// </summary>
     public enum ShaderVariantLogLevel
     {
-        /// <summary>Disable all log for Shader Variant</summary>
+        // <summary>Disable all log for Shader Variant</summary>
```

Comment:
```suggestion
        /// <summary>Disable all log for Shader Variant</summary>
```
The missing `/` make the API doc test fails: 
> Missing Doc on UnityEngine.Rendering.ShaderVariantLogLevel.Disabled